packages/components/table/src/table-body/events-helper.ts

```diff
@@ -8,6 +8,10 @@ import type { TableColumnCtx } from '../table-column/defaults'
 import type { TableBodyProps } from './defaults'
 import type { TableOverflowTooltipOptions } from '../util'
 
+function compareValueWithError(a: number, b: number, error = 0.01) {
+  return a > b && a - b > error
```

Comment:
It seems `a - b > error` is enough. What is the reason why you added two checks?

Reply:
Thank you for your review, I seem to have made unnecessary judgments. 😅